modules/47_regipol/module.gms

```diff
@@ -8,11 +8,26 @@
 
 *' @title Regional Policies
 *'
-*' @description  The 47_regipol module includes region specific policies.
+*' @description  The 47_regipol module includes region-specific policies and adjustments.
```

Comment:
I like this, longer but maybe "..and specific regional adjustments on top of the generic energy system model. "?